DESCRIPTION

```diff
@@ -1,14 +1,15 @@
 Type: Package
 Package: dfeR
 Title: Common DfE R tasks
-Version: 0.5.1
+Version: 0.6.0
 Authors@R: c(
     person("Cam", "Race", , "cameron.race@education.gov.uk", role = c("aut", "cre")),
     person("Laura", "Selby", , "laura.selby@education.gov.uk", role = "aut"),
     person("Adam", "Robinson", role = "aut"),
     person("Jen", "Machin", , "jen.machin@education.gov.uk", role = "ctb"),
     person("Jake", "Tufts", , "jake.tufts@education.gov.uk", role = "ctb"),
-    person("Rich", "Bielby", , "richard.bielby@education.gov.uk", role = "ctb",
+    person("Rich", "Bielby", , "richard.bielby@education.gov.uk", role = "ctb"),
+    person( "Menna", "Zayed", , "menna.zayed@education.gov.uk", role = "ctb",
            comment = c(ORCID = "0000-0001-9070-9969"))
```

Comment:
Think you've now accidentally stolen Rich's ORCID number (it's an academic thing). I think you'll want this instead:

```
    person("Rich", "Bielby", , "richard.bielby@education.gov.uk", role = "ctb",
           comment = c(ORCID = "0000-0001-9070-9969")),
    person( "Menna", "Zayed", , "menna.zayed@education.gov.uk", role = "ctb")
  )
```

There's also a `usethis::use_tidy_description()` function that's nifty and will tidy up any stray spaces / formatting in the description file for you - worth giving that a run to double check it's all 'tidy'

Reply:
my bad! all done :)